data_model/clusters/ACL-Cluster.xml

```diff
@@ -0,0 +1,224 @@
+are the sole property of their respective owners.
+
+This notice and disclaimer must be included on all copies of this document.
+
+Connectivity Standards Alliance
+508 Second Street, Suite 206
+Davis, CA 95616, USA
+-->
+<cluster xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd cluster cluster.xsd" id="0x001f" name="AccessControl" revision="1">
+  <revisionHistory>
+    <revision revision="1" summary="Initial Release"/>
+  </revisionHistory>
+  <classification hierarchy="base" role="utility" picsCode="ACL" scope="Node"/>
+  <dataTypes>
+    <enum name="AccessControlEntryAuthModeEnum">
+      <item value="1" name="PASE" summary="Passcode authenticated session">
+        <mandatoryConform/>
+      </item>
+      <item value="2" name="CASE" summary="Certificate authenticated session">
+        <mandatoryConform/>
+      </item>
+      <item value="3" name="Group" summary="Group authenticated session">
+        <mandatoryConform/>
+      </item>
+    </enum>
+    <enum name="AccessControlEntryPrivilegeEnum">
+      <item value="1" name="View" summary="Can read and observe all (except Access Control Cluster and as seen by a non-Proxy)">
+        <mandatoryConform/>
+      </item>
+      <item value="2" name="Proxy View" summary="Can read and observe all (as seen by a Proxy)">
+        <otherwiseConform>
+          <provisionalConform/>
+          <mandatoryConform/>
+        </otherwiseConform>
+      </item>
+      <item value="3" name="Operate" summary="View privileges, and can perform the primary function of this Node (except Access Control Cluster)">
+        <mandatoryConform/>
+      </item>
+      <item value="4" name="Manage" summary="Operate privileges, and can modify persistent configuration of this Node (except Access Control Cluster)">
+        <mandatoryConform/>
+      </item>
+      <item value="5" name="Administer" summary="Manage privileges, and can observe and modify the Access Control Cluster">
+        <mandatoryConform/>
+      </item>
+    </enum>
+    <enum name="ChangeTypeEnum">
+      <item value="0" name="Changed" summary="Entry or extension was changed">
+        <mandatoryConform/>
+      </item>
+      <item value="1" name="Added" summary="Entry or extension was added">
+        <mandatoryConform/>
+      </item>
+      <item value="2" name="Removed" summary="Entry or extension was removed">
+        <mandatoryConform/>
+      </item>
+    </enum>
+    <struct name="AccessControlEntryStruct">
+      <field id="1" name="Privilege" type="AccessControlEntryPrivilegeEnum">
+        <access fabricSensitive="true"/>
+        <mandatoryConform/>
+      </field>
+      <field id="2" name="AuthMode" type="AccessControlEntryAuthModeEnum">
+        <access fabricSensitive="true"/>
+        <mandatoryConform/>
+      </field>
+      <field id="3" name="Subjects" type="&lt;&lt;ref_DataTypeList&gt;&gt;[&lt;&lt;ref_SubjectId&gt;&gt;]">
```

Comment:
that is a lovely type ....

Reply:
Indeed !

Reply:
https://github.com/csa-data-model/projects/issues/333

Reply:
IMO, even with the borked types, this is a valuable starting point.  I don't know how long it will take to update the scraper, but I would prefer not to block on the scraper fixes for more than a few days; after all, we can put up another PR to fix the shortcomings afterwards

Reply:
I'm tracking these issues in the data model issue, so i'm going to close this comment off here to get the PR in.